admin/tools/docker/deployment/in2p3/examples/backup.sh

```diff
@@ -0,0 +1,3 @@
+set -x
+
+/opt/shmux/bin/shmux -c "sudo -u qserv sh -c 'mkdir -p /qserv/backup/worker-innodb/ && mv /qserv/data/mysql/ib* /qserv/backup/worker-innodb/'" ccqserv{126..149}.in2p3.fr
```

Comment:
I'm not sure how useful is this. We are using per-table InnoDB tablespaces which are stored in per-database directories. The central tablespace that you copy above should not contain anything useful.